proto/proto/dex.proto

```diff
@@ -39,7 +39,7 @@ message SwapClaimBody {
   // Note output for asset 2.
   crypto.NotePayload output_2 = 4;
   // Block root for the associated `Swap` action, identifying the block the `Swap` was included in.
-  crypto.MerkleRoot anchor = 5;
+  // crypto.MerkleRoot anchor = 5;
```

Comment:
how do we identify the block the swap was included in?

Reply:
This was based on the removal of the `anchor` from the `SwapClaimPlan`. The anchor is passed in to the `SwapClaimProof::verify` call: https://github.com/penumbra-zone/penumbra/blob/main/crypto/src/proofs/transparent.rs#L442

Is it possible to get the anchor from the block height? Otherwise I think we'll need to add an `anchor` argument to `SwapClaimPlan::swap_claim_proof` (https://github.com/penumbra-zone/penumbra/blob/main/transaction/src/plan/action/swap_claim.rs#L87). I don't _think_ it requires being in the `SwapClaimBody` for any reason but maybe I'm missing something.